core/modules/utils/dom/modal.js

```diff
@@ -115,7 +116,7 @@ Modal.prototype.display = function(title,options) {
 			attributes: {
 				text: {
 					type: "string",
-					value: title
+					value: tiddler.fields && tiddler.fields.caption || title
```

Comment:
Does this change belong to this PR? What is it good for?

Reply:
Oh, I can use `subtitle` field to do so.

Reply:
Deleted